test/quantization/test_quantize_script.py

```diff
@@ -1133,7 +1107,6 @@ def forward(self, x):
         model = torch.jit.script(M()).eval()
         model = quantize_script(model, qconfig_dict, _test_only_eval_fn, [data], inplace=False, debug=True)
         FileCheck().check_not("quantized::conv2d") \
-                   .check("CallMethod") \
```

Comment:
Why do we not check for CallMethod anymore?

Reply:
yeah, we'll inline the graph in convert so CallMethod from sub module is gone